### ISSUE TITLE
sql/schemachanger: job running status messages are not clear

### ISSUE THREAD

**Issue by ajwerner** [cockroachdb/cockroach#87728]
**Is your feature request related to a problem? Please describe.**
The job running status talks about stages, but does not provide a way to see the current plan. 

**Describe the solution you'd like**
We should probably provide access to the plan (via replanning) and we should probably craft a better message to display about what's going on. 

Jira issue: CRDB-19503

**Comment by postamar** [comment]
This probably should be done before https://github.com/cockroachdb/cockroach/issues/88171

**Comment by ajwerner** [comment]
@RichardJCai let's sync on what to do here.

**Comment by ajwerner** [comment]
At this point, I feel the first step here is to just change the strings to not mention stages and just say `Performing metadata update` or `Performing backfill` or `Performing validation` and then call it a day?

**Comment by ajwerner** [comment]
Maybe we have a (x/y) parenthetical.

**Comment by rafiss** [comment]
closing in favor of https://github.com/cockroachdb/cockroach/issues/125560
